gulpfile.babel.js

```diff
@@ -66,5 +66,5 @@ gulp.task('default', (done) => {
   if (args.production)
     runSequence('server', done);
   else
-    runSequence('server', 'karma-dev', done);
+    runSequence('server', done);
```

Comment:
I personally use phantomjs for unit tests, avoid launching a chrome browser in general, and it also works on Virtual Machines, but test without any browsers would obviously be simpler.